build.gradle

```diff
@@ -16,4 +16,7 @@ dependencies {
     testCompile group: 'cglib', name: 'cglib-nodep', version: '3.2.4'
     testCompile group: 'org.objenesis', name: 'objenesis', version: '2.4'
     testCompile group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.2'
+    // https://mvnrepository.com/artifact/org.slf4j/log4j-over-slf4j
+    testCompile group: 'org.slf4j', name: 'slf4j-log4j12', version: '1.7.22'
```

Comment:
Why do we need this dependency? This might lead to problems with clients using this library, if they don't exclude this dependency explicitly, or will this work without problems, since it's `testCompile`?